client/src/pages/ProfileDetail/BookingForm/BookingForm.tsx

```diff
@@ -0,0 +1,125 @@
+import useStyles from './useStyles';
+import { Box, Typography, Select, MenuItem, OutlinedInput, Grid, Button, Card } from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+import { useFormik } from 'formik';
+
+const bookDetail = {
+  rate: 14,
+};
+const timeSlot = [...Array(24).keys()].map((number) => {
+  if (number == 0) {
+    return {
+      time: '12:00 AM',
+      value: number,
+    };
+  } else if (number < 12) {
+    return {
+      time: number.toString() + ':00 AM',
+      value: number,
+    };
+  } else if (number === 12) {
+    return {
+      time: number.toString() + ':00 PM',
+      value: number,
+    };
+  } else {
+    return {
+      time: (number - 12).toString() + ':00 PM',
+      value: number,
+    };
+  }
+});
+
+const BookingForm = (): JSX.Element => {
+  const classes = useStyles();
+
+  const bookForm = useFormik({
```

Comment:
Good job using Formik here, could we also add in form validation with Yup schemas? Formik has great support for Yup.